src/Forms/Main.cs

```diff
@@ -9496,26 +9503,41 @@ private void MoveFirstWordInNextUp()
                         MakeHistoryForUndo(_language.BeforeLineUpdatedInListView);
                         p.Text = moveUpDown.S1;
                         next.Text = moveUpDown.S2;
-                        SubtitleListview1.SetText(firstIndex, p.Text);
-                        SubtitleListview1.SetText(firstIndex + 1, next.Text);
+                        if (tb == textBoxListViewTextAlternate)
+                        {
+                            SubtitleListview1.SetAlternateText(firstIndex, p.Text);
+                            SubtitleListview1.SetAlternateText(firstIndex + 1, next.Text);
+                        }
+                        else
+                        {
+                            SubtitleListview1.SetText(firstIndex, p.Text);
+                            SubtitleListview1.SetText(firstIndex + 1, next.Text); 
+                        }
                         var selectionStart = textBoxListViewText.SelectionStart;
-                        textBoxListViewText.Text = p.Text;
+                        tb.Text = p.Text;
                         if (selectionStart >= 0)
                         {
-                            textBoxListViewText.SelectionStart = selectionStart;
+                            tb.SelectionStart = selectionStart;
                         }
                     }
                 }
             }
         }
 
-        private void MoveLastWordDown()
+        private void MoveLastWordDown(TextBox tb)
         {
             int firstIndex = FirstSelectedIndex;
             if (firstIndex >= 0)
             {
-                var p = _subtitle.GetParagraphOrDefault(firstIndex);
+                var p = _subtitle.GetParagraphOrDefault(firstIndex); ;

```

Comment:
empty statement ;)

Reply:
Ugh. Sorry. I wonder why Codacy didn't catch it.